vnext/Microsoft.ReactNative/Views/ShadowNodeBase.h

```diff
@@ -128,6 +128,9 @@ struct REACTWINDOWS_EXPORT ShadowNodeBase : public ShadowNode {
   bool m_onMouseEnterRegistered = false;
   bool m_onMouseLeaveRegistered = false;
 
+  // Pointer events
+  std::string m_pointerEvents = "";
```

Comment:
nit: Default value is already empty string

```suggestion
  std::string m_pointerEvents;
```

Reply:
Looking at this further, this would be a bit more efficient to be cached as an enum instead of a string, since we only ever compare it to known values. Consider storing as an enum, then setting accordingly in ViewManagerBase::UpdateProperty. This would also make the ShadowNode a bit smaller, since `std::string` contains a small internal buffer for even empty strings to enable the short string optimization.